spec/requests/api/v2/providers_spec.rb

```diff
@@ -236,6 +240,20 @@
       end
     end
 
+    context "with the maximum number of sites" do
+      let!(:provider) { create(:provider, course_count: 0, site_count: Site::POSSIBLE_CODES.size, organisations: [organisation]) }
```

Comment:
Minor thing, but you don't need `let!` here as the `get` request below already references `provider`. It can also get in the way in the future by creating extra data you don't need.

Reply:
There's a `let!` above and I don't know why

Reply:
It was probably copied from the spec above that one :trollface: